### ISSUE TITLE
[Bug] How to hide or replace element cursor?

### ISSUE THREAD

**Issue by abby0127** [cornerstonejs/cornerstone3D#818]
### Describe the Bug

activate CircleROI tool, no configuration to replace or hide element cursor

### Steps to Reproduce

/

### The current behavior

/

### The expected behavior

/

### OS

Windows 10

### Node version

14.18.0

### Browser

Google Chrome 116.0.5845.142

**Comment by sgielen** [comment]
I dived into the code and found an undocumented feature for disabling this. I have filed #831 for documenting it.

I put the following line before my `cornerstone.init()` call (although I think it works when you do it before setToolActive() too):

```
    cornerstone.Settings.getRuntimeSettings().set('useCursors', false);
```

This ensures the ToolGroup does not activate cursors [here](https://github.com/cornerstonejs/cornerstone3D/blob/e0cd0dfada076421daa18efa66489b3695f84617/packages/tools/src/store/ToolGroupManager/ToolGroup.ts#L365-L366).
